ssaHAL/ssa/_action_handler.py

```
@@ -0,0 +1,251 @@
+class ActDictElement():
+    def __init__(self, callback = None, node_name = None, children = {}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)